### PR TITLE
adds graph to homepage ab#12

### PR DESCRIPTION
**Pull request by MLFigoya**

### CODE REVIEW

**Review by natdarke-figoya — CHANGES_REQUESTED** [review]
Have you got Russia and Japan mixed up?

Is this responsive?

File: index.html [hunk 1]
```diff
@@ -33,6 +33,9 @@ <h2>The Internet pollutes</h2>
               produced by the whole of the UK.
             </p>
           </div>
+          <div>
```
Comment (natdarke-figoya) [inline index.html:36]: Is this responsive?

File: index.html [hunk 2]
```diff
@@ -33,6 +33,9 @@ <h2>The Internet pollutes</h2>
...
+            <img src="@FIGOYA-GRAPHS-24NOV-03.svg">
```
Comment (natdarke-figoya) [inline index.html:37]: Have you got Russia and Japan mixed up

**Review by natdarke-figoya — DISMISSED** [review]

File: index.html (hunk 1 above)
Comment (natdarke-figoya) [inline index.html:36]: Is this responsive?

File: index.html (hunk 2 above)
Comment (natdarke-figoya) [inline index.html]: Have you got Russia and Japan mixed up

**Review by natdarke-figoya — CHANGES_REQUESTED** [review]
Could you have a look my comments please

File: index.html [hunk 3]
```diff
@@ -33,6 +33,9 @@ <h2>The Internet pollutes</h2>
...
+          </div>
```
Comment (natdarke-figoya) [inline index.html:38]: Blah blah. Lulu

**Review by natdarke-figoya — COMMENTED** [review]
I added an comment about Lulu

File: index.html (hunk 2 above)
Comment (MLFigoya) [reply index.html]: yes. I corrected it now

**Pull request merged**